Project.toml

```diff
@@ -13,7 +13,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
-AMDGPU = "0.5"
+AMDGPU = "0.5, 0.6, 0.7"
```

Comment:
Could you please also add the newly released `0.8` and make sure that all works as it should?